lib/view_model/dashboard/dashboard_view_model.dart

```diff
@@ -152,128 +152,18 @@ abstract class DashboardViewModelBase with Store {
         type = appStore.wallet!.type,
         transactions = ObservableList<TransactionListItem>(),
         wallet = appStore.wallet! {
-    name = wallet.name;
-    type = wallet.type;
     isShowFirstYatIntroduction = false;
     isShowSecondYatIntroduction = false;
     isShowThirdYatIntroduction = false;
     updateActions();
 
-    final _wallet = wallet;
-
-    if (_wallet.type == WalletType.monero) {
-      subname = monero!.getCurrentAccount(_wallet).label;
-
-      _onMoneroAccountChangeReaction = reaction(
-          (_) => monero!.getMoneroWalletDetails(wallet).account,
-          (Account account) => _onMoneroAccountChange(_wallet));
-
-      _onMoneroBalanceChangeReaction = reaction(
-          (_) => monero!.getMoneroWalletDetails(wallet).balance,
-          (MoneroBalance balance) => _onMoneroTransactionsUpdate(_wallet));
-
-      final _accountTransactions = _wallet.transactionHistory.transactions.values
-          .where((tx) =>
-              monero!.getTransactionInfoAccountId(tx) == monero!.getCurrentAccount(wallet).id)
-          .toList();
-
-      final sortedTransactions = [..._accountTransactions];
-      sortedTransactions.sort((a, b) => a.date.compareTo(b.date));
```

Comment:
the sorting was removed, can you confirm that it was not needed anyway or in any wallet?